### DISCUSSION TITLE
Is any additional package for system environment necessary?

### DISCUSSION THREAD

**Discussion by Eirinn75** [Romanitho/Winget-AutoUpdate#538] [Q&A] [answered]
When I first tested this package at the beginning of this year, I remember deploying a separate script to allow System access to Winget in its environment. From what I understood this is not necessary anymore, but wanted to be sure. The same should be most probably applicable to the app install scripts "Winget-Install".
Would you suggest to deploy "app installer" from the new microsoft store in order to ensure that all clients are starting with the latest winget version?
Thanks for any clarification

**Comment by Romanitho** [comment, marked as answer]
Just Install WAU. Nothing more. It will install Winget in latest version.
